### PR TITLE
Add visualization for Cluster Agent Operations SLO

### PR DESCRIPTION
**Pull request by Rizwana777**
JIRA LINK - https://issues.redhat.com/browse/GITOPSRVCE-753

Screenshot 2023-10-30 at 5 34 31 PM
Screenshot 2023-10-30 at 5 34 45 PM

### CODE REVIEW

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml [hunk 1]
```diff
@@ -689,67 +864,71 @@ data:
...
+        "regex": "/^rhtap-/",
+        "skipUrlSync": false,
+        "type": "datasource"
+      }
+    ]
+  },
+  "time": {
+    "from": "now-1h",
+    "to": "now"
+  },
```
Comment (amisstea) [inline dashboards/grafana-dashboard-rhtap-slos.configmap.yaml:900]: Please don't modify the overall dashboard settings. We should use the default 28d SLO measurement window.

**Review by amisstea — CHANGES_REQUESTED** [review]
@Rizwana777 please try to minimize the changes to mostly the addition of a panel (see #98 for a good example). The change in indentation also make this difficult to review.

File: dashboards/grafana-dashboard-rhtap-slos.configmap.yaml (hunk 1 above)
Comment (Rizwana777) [reply dashboards/grafana-dashboard-rhtap-slos.configmap.yaml]: Undid modifications in overall dashboard settings

**Comment by Rizwana777** [comment]
> @Rizwana777 please try to minimize the changes to mostly the addition of a panel (see #98 for a good example). The change in indentation also make this difficult to review.

Fixed indentation , PTAL

**Review by amisstea — APPROVED** [review]
Looks much better @Rizwana777, thanks!

@avi-biton @Kousalya1998 PTAL as well.

**Review by avi-biton — APPROVED** [review]
lgtm

**Pull request merged**